emscripten.py

```diff
@@ -929,6 +931,56 @@ def create_invoke_wrappers(invoke_funcs):
   return invoke_wrappers
 
 
+def create_wasm64_wrappers(metadata):
+  # TODO(sbc): Move this into somewhere less static.  Maybe it can become
+  # part of library.js file, even though this metadata relates specifically
+  # to native (non-JS) functions.
+  mapping = {
+    'sbrk': 'pP',
```

Comment:
```suggestion
    'sbrk': 'pp',
```

Reply:
Sorry, I should have documented this sig format.  

The uppercase P here means "optional pointer".     It allows for example `main()` to be called without passing an argv pointer.  With wasm32 this just works.  With wasm64 it will crash because `undefined` is not a valid `BigInt`.

Also, the non-pointer parts of the sig are ignored here so maybe I should just `_`?